modules/AWS/nfs/variables.tf

```diff
@@ -58,9 +52,12 @@ variable "shared_home_snapshot_id" {
     error_message = "Provide correct EBS snapshot ID."
   }
 }
-# The size of the EBS volume to allocate for shared home. If is null no PV and PVC will be created.
+
 variable "shared_home_size" {
-  description = "The storage capacity to allocate to local home"
+  description = "The storage capacity to allocate to shared home"
   type        = string
-  default     = null
+  validation {
+    condition     = can(regex("^[0-9]+([gG]|Gi)$", var.shared_home_size)) || var.shared_home_size == null
```

Comment:
We support only G units due to this line - https://github.com/atlassian-labs/data-center-terraform/pull/217/files#diff-26d8f7b099b5c1c21aeafca0e13eb3676ace8a8bd69e0d240a7446b8cbe8c2efR5

Reply:
Null value for `shared_home` should not be valid